shared-ui/src/commonMain/kotlin/com/tri_tail/ceal_chronicler/ui/title_screen/TitleScreen.kt

```diff
@@ -5,24 +5,32 @@ import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.Image
 import androidx.compose.material.*
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.MutableState
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.semantics.Role.Companion.Image
 import androidx.compose.ui.unit.dp
+import com.tri_tail.ceal_chronicler.MR
 import com.tri_tail.ceal_chronicler.theme.primaryColor
 import com.tri_tail.ceal_chronicler.theme.primaryDarkColor
 import com.tri_tail.ceal_chronicler.theme.typography
 import com.tri_tail.ceal_chronicler.ui.main_view.MainViewState
+import dev.icerock.moko.resources.compose.stringResource
 
 @Composable
 fun TitleScreen(mainViewState: MutableState<MainViewState>) {
     Column(
         verticalArrangement = Arrangement.Center,
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
+        Image(
+            painter = painterResource(id = MR.images.CealChroniclerLogo),

```

Comment:
This is not working yet. Could you please help me figure out how to get this to work?

Reply:
@KiraResari sample added